scanner/src/funTest/kotlin/scanners/FileCounterTest.kt

```diff
@@ -40,18 +40,11 @@ class FileCounterTest : StringSpec() {
     private lateinit var outputRootDir: File
     private lateinit var outputDir: File
 
-    override fun beforeTest(testCase: TestCase) {
-        outputRootDir = createTempDir()
-        outputDir = File(outputRootDir, "output")
-    }
-
-    override fun afterTest(testCase: TestCase, result: TestResult) {
-        outputRootDir.safeDeleteRecursively(force = true)
-        ScanResultsStorage.stats = AccessStatistics()
-    }
-
     init {
-        "Gradle project scan results for a given analyzer result are correct" {
+        "Gradle project scan results for a given analyzer result are correct".config(invocations = 3) {
```

Comment:
Commit message nit: The SHA will change when you rebase and merge this PR, so maybe just say "previous commit"?

Reply:
Ah, right, thanks. I tend to forget that GitHub rewrites commits even if a fast-forward would be possible. Fixed.